src/aiida_quantumespresso/workflows/ph/base.py

```diff
@@ -146,6 +153,40 @@ def prepare_process(self):
             self.ctx.inputs.parameters['INPUTPH']['recover'] = True
             self.ctx.inputs.parent_folder = self.ctx.restart_calc.outputs.remote_folder
 
+    def create_merged_output(self):
+        """Merge outputs from multiple ``PhCalculation`` runs called by the workchain if necessary."""
+        if self.inputs.only_initialization.value:
+            return
+
+        output_dict = {
+            'output_' + str(index + 1): child.outputs.output_parameters
+            for index, child in enumerate(self.ctx.children)
+        }
+
+        num_qpoints = self.ctx.children[0].outputs.output_parameters['number_of_qpoints']
+        num_qpoints_final = self.ctx.children[-1].outputs.output_parameters['num_q_found']
+
+        if num_qpoints == num_qpoints_final:
+            return
```

Comment:
maybe remove this? The logic is if all of the q-points found, then one does not need to merge - however, this is not true for the wall time. 

As we are merging the wall time in other places (or in the case where the q-points are split between files) seems best to do it here as well

Reply:
I've removed this logic. Now the step checks if there is more than one child calculation for the restart work chain.